site/index.html

```diff
@@ -327,7 +327,211 @@ <h5>When</h5>
         <div class="container">
           <h6>Event Schedule</h6>
           <h3>Conference Schedule</h3>
-          <p>Coming soon...</p>
+          <div class="tabs-custom tabs-horizontal tabs-corporate">
+            <!--Tab panes-->
+            <div class="tab-content wow fadeIn">
+              <div class="tab-pane">
+                <div class="card-group-custom card-group-corporate" id="accordion1" role="tablist" aria-multiselectable="false">
+                                    <div class="schedule-classic schedule-classic-simple">
+                                      <div class="schedule-classic-content"><span class="schedule-classic-time">11:00 am to 11:05 am</span><span class="schedule-classic-title heading-4">Welcome!</span></div>
+                                    </div>
+                  <!--Bootstrap card-->
+                  <article class="card card-custom card-corporate">
+                    <div class="card-header" role="tab">
+                      <div class="card-title"><a class="collapsed" id="accordion1-card-head-hfhsoskt" data-toggle="collapse" data-parent="#accordion1" href="#accordion1-card-body-sartmhlo" aria-controls="accordion1-card-body-sartmhlo" aria-expanded="false" role="button"><span class="schedule-classic"><span class="unit unit-spacing-md align-items-center d-block d-md-flex"><span class="unit-left"><span class="schedule-classic-img"><img src="images/speakers/paul-taylor.png" alt=""/></span></span><span class="unit-body"><span class="schedule-classic-content"><span class="schedule-classic-time">11:00 am to 11:40 am (UTC)</span><span class="schedule-classic-title heading-4">Unhappy Path & Dealing with Bad Events</span><span class="schedule-classic-author">by <span class="schedule-classic-author-name">Paul Taylor</span> -  Strategic Advisor at Mulesoft</span></span></span></span></span></a></div>
```

Comment:
```suggestion
                      <div class="card-title"><a class="collapsed" id="accordion1-card-head-hfhsoskt" data-toggle="collapse" data-parent="#accordion1" href="#accordion1-card-body-sartmhlo" aria-controls="accordion1-card-body-sartmhlo" aria-expanded="false" role="button"><span class="schedule-classic"><span class="unit unit-spacing-md align-items-center d-block d-md-flex"><span class="unit-left"><span class="schedule-classic-img"><img src="images/speakers/paul-taylor.png" alt=""/></span></span><span class="unit-body"><span class="schedule-classic-content"><span class="schedule-classic-time">11:10 am to 11:40 am (UTC)</span><span class="schedule-classic-title heading-4">Unhappy Path & Dealing with Bad Events</span><span class="schedule-classic-author">by <span class="schedule-classic-author-name">Paul Taylor</span> -  Strategic Advisor at Mulesoft</span></span></span></span></span></a></div>
```

Reply:
I cut initial proposal time to 30 like all the others, so this one should start at 11.10